src/main/java/software/amazon/cloudwatchlogs/emf/serializers/StorageResolutionSerializer.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package software.amazon.cloudwatchlogs.emf.serializers;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import java.io.IOException;
+import software.amazon.cloudwatchlogs.emf.model.StorageResolution;
+
+public class StorageResolutionSerializer extends StdSerializer<StorageResolution> {
+    StorageResolutionSerializer() {
```

Comment:
What is this for?

Reply:
From what I observed, when the code attempts to serialize, it reaches this [line](https://github.com/awslabs/aws-embedded-metrics-java/blob/master/src/main/java/software/amazon/cloudwatchlogs/emf/model/RootNode.java#L92), which calls a method of `ObjectMapper` class of `com.fasterxml.jackson.databind` package. This internally calls `createInstance()` method of `ClassUtil` class of `com.fasterxml.jackson.databind.util` package which instantiates  `StorageResolutionSerializer` looking for a default no-argument constructor (the one you highlighted int he comment) in the class. 

So if I remove this constructor, multiple tests related to serialization fails. Also, similar behaviour can be observed in [UnitSerializer](https://github.com/awslabs/aws-embedded-metrics-java/blob/master/src/main/java/software/amazon/cloudwatchlogs/emf/serializers/UnitSerializer.java#L27) class.